.github/workflows/main.yml

```diff
@@ -2,11 +2,12 @@ on: [push, pull_request]
 
 jobs:
   check-packages:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-24.04
```

Comment:
This doesn't look correct to me (here and throughout this file). Changing this from `ubuntu-latest` (which basically always should work) to `ubuntu-24.04` presumably puts us on a never-ending tech-debt treadmill that forces us to update this every few years. Am I wrong about that?

Reply:
looks like this comes from: https://github.com/delphix/delphix-platform/pull/502

Reply:
I could see it going both ways.. i.e. if we use latest, then things can break without us due to the image changing.. if we use a pinned version, things should remain stable as the image doesn't change, but we have to play the game of updating it every-so-often.. I'm good changing it back to latest.. think that should be fine..